src/BenchmarkDotNet/Parameters/ParameterInstance.cs

```diff
@@ -55,8 +55,15 @@ public string ToDisplayText(CultureInfo cultureInfo)
             }
         }
 
+        public string ToDisplayText(SummaryStyle summary)
+        {
+            return summary != null ? ToDisplayText(summary.CultureInfo, summary.MaxParameterColumnWidth) : ToDisplayText();
+        }
+
         public string ToDisplayText() => ToDisplayText(CultureInfo.CurrentCulture);
 
+        private string ToDisplayText(CultureInfo cultureInfo) => ToDisplayText(cultureInfo, maxParameterColumnWidthFromConfig);
+
```

Comment:
is there only one usage of `ToDisplayText(CultureInfo cultureInfo)`? If so, can we inline it?


```suggestion

        public string ToDisplayText() => ToDisplayText(cultureInfo, maxParameterColumnWidthFromConfig);
```

Reply:
👍